### ISSUE TITLE
OutOfMemoryError immediately after starting to encode

### ISSUE THREAD

**Issue by mrwnwttk** [master-of-zen/Av1an#53]
I'm trying to run the following command:
`python av1an.py --input_file input.mkv -enc aom -w 16 -v " --cq-level=30 --end-usage=q --aq-mode=2 " -ff " -colorspace bt2020nc -color_trc smpte2086 -color_primaries bt2020 " -log log.txt --output_file output.mkv`

But immediately after the script starts running I get the following error message (after frame 3 of 15992):
`19:52:23 Starting scene detection Threshold: 50
19:52:24 Error in PySceneDetect: OpenCV(4.2.0) C:\projects\opencv-python\opencv\modules\core\src\alloc.cpp:73: error: (-4:Insufficient memory) Failed to allocate 24883200 bytes in function 'cv::OutOfMemoryError'`

Sometimes it happens after a single frame, sometimes after seven. It seems that the error occurs right as the first worker wants to allocate memory (somewhere in scene_detect it throws the exception).

I'm certain that I'm not out of memory as I still had 25/32GB of RAM left and 24883200 is only around 24MB.

I've installed the requirements according to the requirements.txt, all the needed executables are on the PATH and also in the folder that the python script is in. I'm running Windows 10 in case that makes a difference.

Did I not configure something properly?

**Comment by master-of-zen** [comment]
It's first time i see that issue with OpenCV.
Is it appeared right after install or you had encodes that were ok before that?
Is it persist on other videos? Can suggest video for test
Try installing Av1an from PIP and use `av1an -i file`

**Comment by mrwnwttk** [comment]
This is the first video I've tried to encode. It's 2160p HDR10 x265, which might have played a role. I just tried a different file (1080p x264 SDR) without the ffmpeg parameters and this one encodes just fine, as long as it's normal for my CPU (Ryzen 7 3700X 8C/16T) to be maxed out at 100% constantly with progress at 0 frames (and the log reporting that it started 16 workers with around 200 frames each).  I'm not sure whether it was the HDR file or just the ffmpeg parameters that threw av1an off, will do some more testing and report back.
